### PR TITLE
[Arm64] EXTERNAL_FeatureSIMD_Default = 1

### PR DESCRIPTION
**Pull request by sdmaclea**
I expect this to be the last SIMD patch to merge in the initial SIMD bring up

### CODE REVIEW

File: src/inc/clrconfigvalues.h [hunk 1]
```diff
@@ -545,13 +545,16 @@ CONFIG_DWORD_INFO_EX(INTERNAL_JitLoopHoistStats, W("JitLoopHoistStats"), 0, "Dis
 CONFIG_DWORD_INFO_EX(INTERNAL_JitDebugLogLoopCloning, W("JitDebugLogLoopCloning"), 0, "In debug builds log places where loop cloning optimizations are performed on the fast path.", CLRConfig::REGUTIL_default);
 CONFIG_DWORD_INFO_EX(INTERNAL_JitVNMapSelLimit, W("JitVNMapSelLimit"), 0, "If non-zero, assert if # of VNF_MapSelect applications considered reaches this", CLRConfig::REGUTIL_default)
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_JitVNMapSelBudget, W("JitVNMapSelBudget"), 100, "Max # of MapSelect's considered for a particular top-level invocation.")
-#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
+#if defined(_TARGET_XARCH_) || defined(_TARGET_ARM64_)
```
Comment (BruceForstall) [inline src/inc/clrconfigvalues.h]: I don't think you can (or should?) use `_TARGET_XARCH_` in this file.

**Comment by sdmaclea** [comment]
Is the set of SIMD intrinsics complete?  If others are added they may not always be available.  The crypto (sha, crc...) instructions I believe are optional.  Also ARM is adding extensions so new instructions might be added.

**Comment by sdmaclea** [comment]
@dotnet-bot test Windows_NT arm64 Cross Checked jitstress1 Build and Test
@dotnet-bot test Windows_NT arm64 Cross Checked jitstress2 Build and Test

**Comment by sdmaclea** [comment]
> One larger question: should SIMD code generation be enabled on ARM64 for precompiled (NGEN/crossgen) code?

@CarolEidt comments?

**Comment by sdmaclea** [comment]
@BruceForstall 
+ Reverted x86 + x64 => xarch per your request.  
+ Removed WIP No merge from title
+ Cherry-picked the latest enable SIMD compilation into this PR too 
+ Rebased

**Comment by sdmaclea** [comment]
I did not enable SIMD during crossgen (because I never took the time to figure out how).  However,  coverage is bad with crossgen as it is rarely run in CI.  So it seems better to keep SIMD disabled in crossgen initially.

**Comment by sdmaclea** [comment]
Merging this change into #14633

**Pull request closed without merging**